backend/app/src/main/java/com/moh/phlat/backend/esb/json/PlrResponse.java

```diff
@@ -0,0 +1,14 @@
+package com.moh.phlat.backend.esb.json;
+
+import java.text.SimpleDateFormat;
+
+import com.moh.phlat.backend.esb.boundary.PlrToken;
+import com.moh.phlat.backend.model.ProcessData;
+
+public interface PlrResponse {
+	public static final SimpleDateFormat JSON_DATE_FORMAT_OJDK11 = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss");
+	public void plrJsonToProcessData(String json, ProcessData data);
+	public void handlePlrError(Exception ex);
+	public void handleKeyCloakError(PlrToken token);
```

Comment:
We cannot proceed if we are encountering Keycloak issues. What is the purpose of including it in the list of errors?

Reply:
As written, the Retry and CodeBreaker annotations should catch and handle this. I haven't tested this yet, but I'll include it in my testing against Confirmation today